spec/components/candidate_interface/application_status_tag_component_spec.rb

```diff
@@ -17,6 +17,47 @@
       end
     end
 
+    context 'when the application choice is in the `interviewing` state' do
+      it 'tells the candidate when the reject by default date will be' do
+        application_choice = create(
+          :application_choice,
+          :interviewing,
+          reject_by_default_at: 5.days.from_now,
+        )
+        result = render_inline(described_class.new(application_choice: application_choice))
+
+        expect(result.text).to include(
+          "You'll get a decision on your application by #{5.days.from_now.to_s(:govuk_date)}.",
+        )
+      end
+    end
+
+    context 'when the application choice is in the `awaiting_provider_decision` state' do
+      it 'tells the candidate when the reject by default date will be' do
+        application_choice = create(
+          :application_choice,
+          :awaiting_provider_decision,
+          reject_by_default_at: 14.days.from_now,
+        )
+        result = render_inline(described_class.new(application_choice: application_choice))
+
+        expect(result.text).to include(
+          "You'll get a decision on your application by #{14.days.from_now.to_s(:govuk_date)}.",
+        )
+      end
+
+      it 'handles nil values for `reject_by_default_at`' do
+        application_choice = create(
+          :application_choice,
+          :awaiting_provider_decision,
+          reject_by_default_at: nil,
+        )
+        result = render_inline(described_class.new(application_choice: application_choice))
+
+        expect(result.text).not_to include('You'll get a decision on your application by')
```

Comment:
Wonder if the use of a ' here within the sentence might break the tests that are running

Reply:
Thanks @MylesJarvis. The `'` would need to be prefixed with a `\` to avoid confusing the Ruby interpreter but it should really be a `’` anyway. Changed in later commit.